fusor-ember-cli/app/controllers/review/installation.js

```diff
@@ -242,33 +242,18 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, OpenshiftMixin, {
     return this.get('storageSize') + ' GB';
   }),
 
-  deploymentButtonAction: Ember.computed('hasSubscriptionsToAttach', function() {
-    if (this.get('showWarningMessage')) {
-        return "showContinueDeployModal";
-    } else if (this.get('hasSubscriptionsToAttach')) {
-        return "attachSubscriptions";
-        return "installDeployment";
-    } else if (this.get('showWarningMessage')) {
-        return "showContinueDeployModal";
-    }
-  }),
-
   closeContinueDeployModal() {
-    this.set('continueDeploymentModalOpen', false);
-    this.set('continueDeploymentModalClosed', true);
-    this.set('modalOpen', false);
+    this.set('openModal', false);
   },
 
   actions: {
     showContinueDeployModal() {
       this.set('deploymentInModal', this.get('model'));
-      this.set('continueDeploymentModalOpen', true);
-      this.set('continueDeploymentModalClosed', false);
-      this.set('modalOpen', true);
+      this.set('openModal', true);
     },
 
     onDeployButton() {
-      if (this.get('showWarningMessage')) {
+      if (this.get('showValidationWarnings') || this.get('showValidationErrors')) {
```

Comment:
@cfchase, @eriknelson, there was already a computed property for showValidationWarnings so I'm using this. Should I also include if this.get('showValidationErrors') to bring up the continue-deploy-modal? I'm not sure.